### PR TITLE
24759 - Update Alteration Warning Language

### PR DESCRIPTION
**Pull request by ArwenQin**
*Issue #:* /bcgov/entity#24759
*Description of changes:*
- Update the language in "Change company name" warning

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by ArwenQin** [comment]
After update:
image

File: src/components/common/YourCompany/ChangeBusinessType.vue [hunk 1]
```diff
@@ -173,8 +173,15 @@
             <strong class="pl-2 gray9--text text-small-text">Change company name</strong>
           </header>
           <article class="pl-8 pt-1 small-text">
-            To change to a {{ GetCorpFullDescription(selectedEntityType) }}, you must change the company
-            name using an approved name request or change it to a numbered company.
+            To change to a {{ GetCorpFullDescription(selectedEntityType) }}, you must change your company name.
+            You can use your incorporation number as the company name or go to
+            <a
+              href="https://dev.names.bcregistry.gov.bc.ca/"
+              class="underline-blue-text"
```
Comment (ArwenQin) [inline src/components/common/YourCompany/ChangeBusinessType.vue:180]: Shall we add the url to the .env file?

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-edit-dev--pr-604-kq7m7io7.web.app

File: src/components/common/YourCompany/ChangeBusinessType.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/common/YourCompany/ChangeBusinessType.vue:180]: Yes!

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-edit-dev--pr-604-kq7m7io7.web.app

File: src/components/common/YourCompany/ChangeBusinessType.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/common/YourCompany/ChangeBusinessType.vue:180]: Does this link open in a new window/tab? Should it?

File: .env.example [hunk 2]
```diff
@@ -6,6 +6,7 @@ VUE_APP_ADDRESS_COMPLETE_KEY=
 
 #vaults web-url
 VUE_APP_REGISTRY_HOME_URL="https://dev.bcregistry.gov.bc.ca/"
+VUE_APP_REGISTRY_NAME_REQUEST_URL="https://dev.names.bcregistry.gov.bc.ca/"
```
Comment (severinbeauvais) [inline .env.example:9]: Remove "REGISTRY" from this name.

**Review by severinbeauvais — APPROVED** [review]
LGTM

**Review by AimeeGao — APPROVED** [review]
Apart from Sev's comments, everything looks good to me (^-^)

**Review by meawong — APPROVED** [review]
Tested and LGTM! Approved once Sev's comments are addressed.

**Comment by ArwenQin** [comment]
Added a new icon, per updated design:
image

File: devops/vaults.env [hunk 3]
```diff
@@ -6,6 +6,7 @@ VUE_APP_ADDRESS_COMPLETE_KEY="op://canadapost/$APP_ENV/address-key/ADDRESS_COMPL
 
 #vaults web-url
 VUE_APP_REGISTRY_HOME_URL="op://web-url/$APP_ENV/registry/REGISTRY_HOME_URL"
+VUE_APP_REGISTRY_NAME_REQUEST_URL="op://web-url/$APP_ENV/registry/REGISTRY_NAME_REQUEST_URL"
```
[the remark this answers is not in the document]
Comment (ArwenQin) [reply devops/vaults.env]: Updated

File: src/components/common/YourCompany/ChangeBusinessType.vue [hunk 4]
```diff
@@ -173,8 +173,21 @@
...
-            name using an approved name request or change it to a numbered company.
+            To change to a {{ GetCorpFullDescription(selectedEntityType) }}, you must change your company name.
+            You can use your incorporation number as the company name or go to
+            <a
+              :href="nameRequestUrl"
+              class="underline-blue-text"
+              target="_blank"
+              rel="noopener"
+            >
+              Name Request
```
Comment (ArwenQin) [inline src/components/common/YourCompany/ChangeBusinessType.vue:184]: Open in a new tab. Does this method look ok?

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-edit-dev--pr-604-kq7m7io7.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-edit-dev--pr-604-kq7m7io7.web.app

**Review by severinbeauvais — APPROVED** [review]
Looks all-good to me.

**Pull request merged**